lib/src/KPIs/index.tsx

```diff
@@ -1,24 +1,23 @@
 import React from 'react';
 
+import { IoOpenOutline } from 'react-icons/io5';
 import styles from './styles.module.css';
-import externalLink from './external-link.svg';
 
 interface KpiData {
-    title: string;
-    subtitle: string;
-    value: React.ReactNode;
-    date: string;
-    source: string;
-    backgroundColor: string;
-    color: string;
-    primaryColor: string;
-    url: string;
+    title?: string;
+    titleStyle: React.CSSProperties;
+    subtitle?: string;
+    subtitleStyle: React.CSSProperties;
+    value?: number;
+    valueStyle: React.CSSProperties;
+    date?: string;
+    source?: string;
+    url?: string;
+    sourceStyle: React.CSSProperties;
```

Comment:
We should not use React.CSSProperties. Instead we should define our own configuration.